src/install-ruby.ts

```diff
@@ -0,0 +1,51 @@
+import createDebug from 'debug';
+import execa from 'execa';
+
+const debug = createDebug('@zeit/fun:install-ruby');
+
+export async function installRVM(version: string): Promise<any> {
+	try {
+		await execa('rvm');
+		debug('RVM present');
+	} catch (error) {
+		debug(error);
+		debug('Installing RVM');
+		execa.command('curl -sSL https://get.rvm.io | bash -s stable');
```

Comment:
Can we install to a temp directory? We don't want to permanently install something when the user runs `now dev` that would affect the PATH outside of `now dev` processes.